kiwi/xml_description.py

```diff
@@ -108,6 +112,25 @@ def load(self):
             raise KiwiDescriptionInvalid(message)
         return self._parse()
 
+    def _get_validation_details(self):
+        """
+        Run jing process to validate description against the schema
+
+        Jing command provides detailed feedback in case of schema
+        validation failure
+        """
+        try:
+            cmd = Command.run(
+                ['jing', Defaults.get_schema_file(), self.description_xslt_processed.name],
```

Comment:
sounds good to me, you can raise a KiwiCommandNotFound exception from the Command class and catch that in your code. so all others benefit too, that's nice